gtsam/geometry/SOn.h

```diff
@@ -54,7 +54,7 @@ class SO : public LieGroup<SO<N>, internal::DimensionSO(N)> {
   using VectorN2 = Eigen::Matrix<double, internal::NSquaredSO(N), 1>;
   using MatrixDD = Eigen::Matrix<double, dimension, dimension>;
 
-  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+  GTSAM_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
Should we then replace *all* these??

Reply:
Yeah, this is still RFC so I didn't replace everything. I can make another PR that does this replacement after this is merged.

Reply:
See my comment below about branch from develop. I propose that in the new PR you replace all...

Reply:
Sure, will do.